app/questionnaire/questionnaire_store_updater.py

```diff
@@ -97,6 +97,10 @@ def remove_answers(self, answer_ids: List):
         for answer_id in answer_ids:
             self._answer_store.remove_answer(answer_id)
 
+    def remove_answers_with_list_item_id(self, answer_ids: List, list_item_id: str):
+        for answer_id in answer_ids:
+            self._answer_store.remove_answer(answer_id, list_item_id)
+
```

Comment:
A new method is not needed, I think remove answer should just be updated to also take a list_item_id which default to None like:
```py
def remove_answers(self, answer_ids: List, list_item_id: str = None):
    for answer_id in answer_ids:
        self._answer_store.remove_answer(answer_id, list_item_id)
```

Reply:
Removed extra method now, thanks!